### DISCUSSION TITLE
Exclude specific folders when using default "all photos" with no --album

### DISCUSSION THREAD

**Discussion by vorwd** [icloud-photos-downloader/icloud_photos_downloader#553] [Ideas]
Hello -- firstly, great solution and very happy overall, so thank you!

I have searched for other comments and didn't find this mentioned.  Is there a way currently, or in the future, to exclude photos/videos in a certain album when letting icloudpd do "all photos" default?  In particular, I am trying to exclude the hidden folder, which I download with a separate batch file into a different location than "all photos" directory.  

Thanks in advance.

**Comment by AndreyNikiforov** [comment]
IIUC Albums in iCloud are just organized links to All Photos collection. When you go to iCloud.com, do you see photos from your hidden album in All Photos?

`icloudpd` downloads `All Photos` and have a separate command to download one album at a time. There are ways to limit photos downloaded from 'All Photos' collection by asset type (e.g. exclude live photos), but no other filters.

**Reply by burnoutberni** [reply]
I'd like to follow-up on this one: Do you think it's feasible to add a filter like this, so I can exclude photos in a certain album from being processed by `icloudpd`?

I use `icloudpd` to transfer all my photos to another cloud daily and use the `move` mode, so they are all deleted from iCloud (and thus also from my iPhone). This works really well (thank you!) and keeps both my local and iCloud storage cleared.

There are some photos I'd like to keep locally on my iPhone though, e.g. photos I use for wallpapers. My current workaround is to manually move photos to a specific album (`Upload`) and I've configured `icloudpd` to only use this as an input. It would be much more convenient though to go the other way around and upload all photos _except_ the ones I keep in specific albums (e.g. `Wallpapers`).

**Reply by burnoutberni** [reply]
Ah, I think someone else recently opened an issue with the same suggestion: #999
